### ISSUE TITLE
Component Button

### ISSUE THREAD

**Issue by hackos123** [hackos123/ui_components_stack#2]
Show your skills and create customized buttons and add it to the buttons folder
For Example:
``` buttons/<username>```
After creating the component raise a merge request mentioning this issue

**Comment by roy-herzlichkeit** [comment]
Greetings, I'm participating in Hacktoberfest 2024 and new to Open Sourcing, I'm interested in creating customized buttons, Could you please elaborate the task more and assign it to me?

**Comment by hackos123** [comment]
Greetings @cordiality-crypto !!!
This repository is related to creating custom UI components.
We are working on the Website were this components would be displayed and the developers can freely access this components.

This task is regarding custom responsive button  component.
You can start working on any creative design related to a button which comes into your mind but please don't copy and paste code from anywhere else.

**Comment by roy-herzlichkeit** [comment]
Okay, got it! Thanks, @hackos123 for assigning.

**Comment by roy-herzlichkeit** [comment]
@hackos123 I created a PR, please assure my code and add the label. Thanks!

**Comment by biswajit-sarkar-007** [comment]
Hello! I'm interested in this issue. Can I be assigned?

**Comment by hackos123** [comment]
@biswajit-sarkar-007 yes you can start working on this component. as this is a small component try creating multiple variations in the same pull request.

**Comment by biswajit-sarkar-007** [comment]
@hackos123 , please check out the [pull request #11](https://github.com/hackos123/ui_components_stack/pull/11) for the latest updates. Your feedback would be greatly appreciated!

**Comment by biswajit-sarkar-007** [comment]
@hackos123 , please review the changes in [PR #11] (https://github.com/hackos123/ui_components_stack/pull/11).

**Comment by biswajit-sarkar-007** [comment]
@username, please review the changes in [pull request #11 ] (https://github.com/hackos123/ui_components_stack/pull/11).

**Comment by hackos123** [comment]
@biswajit-sarkar-007 it is reviewed!!!
keep contributing!!!
